MoneyManagerModel/User.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System.Collections.Generic;
+
+namespace MoneyManagerModel
+{
+    public class User
+    {
+        public int 
+            UserId { get; set; }
+
+        public string 
+            Name { get; set; }
+
+        public string 
+            Email { get; set; }
+
+        public ICollection<Asset> 
+            Assets { get; set; }
+
+        public User(
+            string name, 
+            string email)
+        {
+            Name = name;
+            Email = email;
+        }
+
+        public User() { }
```

Comment:
The constructor will be generated automatically by CLR, so its declaration can be omitted.